rotkehlchen/tests/api/blockchain/test_avalanche.py

```diff
@@ -71,7 +71,7 @@ def test_add_avax_blockchain_account(rotkehlchen_api_server):
     result = assert_proper_response_with_result(response)
 
     # Check per account
-    account_balances = result['per_account']['AVAX'][AVALANCHE_ACC1_AVAX_ADDR]
+    account_balances = result['per_account']['avax'][AVALANCHE_ACC1_AVAX_ADDR]
     assert 'liabilities' in account_balances
     asset_avax = account_balances['assets']['AVAX']
```

Comment:
why is is it still uppercase here? The right approach would be to also use `xxx.serialize()` in the tests